package proto_ff;

import "yd_fieldoptions.proto";


message teleportborn
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "点ID"];
	optional int32 type = 2[(yd_fieldoptions.field_cname) = "点类型"];
	optional int64 scene = 3[(yd_fieldoptions.field_cname) = "所属场景"];
	optional int32 roleBornRange = 4[(yd_fieldoptions.field_cname) = "角色出生范围"];
}


message Sheet_teleportborn
{
	repeated teleportborn teleportborn_List = 1[(yd_fieldoptions.field_arysize)=300];
}

message teleportteleport
{
	optional int64 teleportID = 1[(yd_fieldoptions.field_cname) = "传送点ID"];
	optional int32 teleportType = 2[(yd_fieldoptions.field_cname) = "传送门类型"];
	optional string name = 3[(yd_fieldoptions.field_cname) = "传送点名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int64 monsterId = 4[(yd_fieldoptions.field_cname) = "传送门怪物ID"];
	optional int32 showCondition = 5[(yd_fieldoptions.field_cname) = "出现条件"];
	optional string conditionValue = 6[(yd_fieldoptions.field_cname) = "条件参数", (yd_fieldoptions.field_bufsize) = 80];
	optional int32 canEnter = 7[(yd_fieldoptions.field_cname) = "进入权限"];
	optional int64 showPathID = 8[(yd_fieldoptions.field_cname) = "默认位置点"];
	optional int64 toPathID = 9[(yd_fieldoptions.field_cname) = "传送目的位置点"];
	optional int64 toDuplicateId = 10[(yd_fieldoptions.field_cname) = "传送副本ID"];
	optional string enterCost = 11[(yd_fieldoptions.field_cname) = "进入道具消耗", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 trigRange = 12[(yd_fieldoptions.field_cname) = "触发范围"];
	optional int32 livingTime = 13[(yd_fieldoptions.field_cname) = "传送门存在时间"];
	optional int64 openChatId = 14[(yd_fieldoptions.field_cname) = "开启时的跑马灯ID"];
	optional int64 progressChatId = 15[(yd_fieldoptions.field_cname) = "进度跑马灯ID"];
}


message Sheet_teleportteleport
{
	repeated teleportteleport teleportteleport_List = 1[(yd_fieldoptions.field_arysize)=40];
}
